Triphouse-project/style.css

```diff
@@ -115,6 +131,7 @@
   font-size: 50px;
   line-height: 60px;
   margin-bottom: 100px;
+  padding-right: 650px;
```

Comment:
![2021-04-26_21h04_02](https://user-images.githubusercontent.com/19935417/116129680-0d840c00-a6d3-11eb-937e-1f790c750f5c.png)
What is sens ?

Reply:
fix

Reply:
fix